Dockerfile

```diff
@@ -1,9 +1,11 @@
-ARG PYTHON_VERSION=3.12
+FROM python:3.12.10-slim@sha256:85824326bc4ae27a1abb5bc0dd9e08847aa5fe73d8afb593b1b45b7cb4180f57 AS app_base
 
-FROM python:${PYTHON_VERSION}-slim AS app_base
+# Set up user and group
+ARG groupid=10001
+ARG userid=10001
```

Comment:
The way this local dev environment works, it runs everything from the image and the image gets rebuilt if things change. The other way we've done this is by mounting the repository directory into the image. In that model, for Linux users, the user id and group id need to match the local user. I figured I'd set up part of that here.